src/log/EntityDump.cpp

```diff
@@ -0,0 +1,99 @@
+#include "EntityDump.h"
+#include <iostream>
+#include "../buildings/DropPoint.h"
+
+
+/*Created by Maitiu Morton 4/3/2017
+ *Loops through each manager and writes there information to
+ *the EntityDumpLog.txt
+ */
+void dumpEntityPositions(const Player* p){
+    #ifndef NDEBUG
+    std::ofstream entityDump;
+    std::pair<float, float> coord;
+    GameManager *gm = GameManager::instance();
+
+
+    entityDump.open("EntityDumpLog.txt");
+
+    entityDump << "ENTITY POSITION DUMP\n\n";
+
+    coord  = getCoordinates(dynamic_cast<Entity*>(p->getMarine()));
+    entityDump << "PLAYER MARINE'S POSITION X:" << coord.first << " Y:" << coord.second
+     << std::endl << std::endl;
```

Comment:
Don't use endl as it flushes the stream and I doubt that's what you want judging by how you've used it twice in a row here
It's better to simply write `\n` and flush explicitly